tests/html/multiparser/unknown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,71 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`unknow-lang.html format 1`] = `
+====================================options=====================================
+parsers: ["html"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <style lang="unknown">
+foo
+       bar
+</style>
+
+<script lang="unknown">
+foo
+       bar
+</script>
+
+<script type="unknown">
+foo
+       bar
+</script>
+
+<script type="unknown" lang="unknown">
+foo
+       bar
+</script>
+</body>
+</html>
+
+=====================================output=====================================
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <style lang="unknown">
+      foo
+             bar
```

Comment:
Maybe we should keep the indention too?

@thorn0

Reply:
Probably. It's safer.

Reply:
I think adding indentation for whole block is good idea

Reply:
We don't know any style languages that can be broken by it, do we? So practically speaking, adding indentation is okay.

Reply:
Okay